lib/modules/hideChildComments.js

```diff
@@ -118,3 +118,9 @@ function addToggleChildrenButton(comment) {
 	li.appendChild(a);
 	menu.appendChild(li);
 }
+
+export function toggle(thing: *) {
+	const button = thing.entry.querySelector('a.toggleChildren');
+	if (!button) throw new Error('Toggle button not found');
+	button.click();
```

Comment:
It would be nice to see this module refactored to avoid the click, but this is a reasonable compromise.